internal/storage/fake/testing/bucket_tests.go

```diff
@@ -543,15 +543,35 @@ func (t *createTest) ObjectAttributes_Default() {
 	ExpectThat(o.Deleted, timeutil.TimeEq(time.Time{}))
 	ExpectThat(o.Updated, t.matchesStartTime(createTime))
 
-	// Make sure it matches what is in a listing.
-	listing, err := t.bucket.ListObjects(t.ctx, &gcs.ListObjectsRequest{})
+	// Make sure it matches when we stat object.
+	minObj, extendedAttr, err := t.bucket.StatObject(
+		t.ctx,
+		&gcs.StatObjectRequest{Name: o.Name,
+			ForceFetchFromGcs:              true,
+			ReturnExtendedObjectAttributes: true})
 	AssertEq(nil, err)
 
-	AssertThat(listing.CollapsedRuns, ElementsAre())
-	AssertEq("", listing.ContinuationToken)
-
-	AssertEq(1, len(listing.Objects))
-	ExpectThat(listing.Objects[0], DeepEquals(o))
+	ExpectThat(minObj.Name, Equals(o.Name))
```

Comment:
can you extract this into private methods - comparing minObject and extendedObjectAttributes

Reply:
Updated.